library/Haste/Form/Form.php

```diff
@@ -478,7 +478,22 @@ function($varValue, \Widget $objWidget, Form $objForm) use ($arrDca, $strName) {
             );
         }
 
-        $arrDca = $strClass::getAttributesFromDca($arrDca, $arrDca['name'], $arrDca['value']);
+        $objDca = null;
+        $strTable = '';
+
+        if (null !== ($objModel = $this->getBoundModel())) {
+            $strTable = $objModel->getTable();
+            \Contao\Controller::loadDataContainer($strTable);
+
+            if (isset($GLOBALS['TL_DCA'][$strTable]) && isset($GLOBALS['TL_DCA'][$strTable]['config']['dataContainer'])) {
+                $dataContainer = '\Contao\DC_' . $GLOBALS['TL_DCA'][$strTable]['config']['dataContainer'];
```

Comment:
I think the driver must be in the global namespace. Pretty sure our `DC_Multilingual` is not in the Contao namespace.

Reply:
Right, changed :)